app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -447,8 +447,9 @@ public Study createStudy(Study study) {
         
         emailVerificationService.verifyEmailAddress(study.getSupportEmail());
 
-        sendVerifyEmail(study, StudyEmailType.CONSENT_NOTIFICATION);
-
+        if (study.getConsentNotificationEmail() != null) {
+            sendVerifyEmail(study, StudyEmailType.CONSENT_NOTIFICATION);    
+        }
```

Comment:
Can we make a unit test that creates a study without a consent notification email, so that we can test this branch?

Reply:
Added to the mock test.